### ISSUE TITLE
Multiple async calls to get token returns only one response

### ISSUE THREAD

**Issue by zoperohan** [AzureAD/microsoft-authentication-library-for-js#1066]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## Browser:
- [x] Chrome version 77.0.3865.120
- [ ] Firefox version XX
- [ ] IE version XX
- [ ] Edge version XX
- [ ] Safari version XX

## Library version

<pre><code>
Library version: msal v1.1.3 2019-08-14

</code></pre>

## Current behavior

multiple async calls made to msal.js to get tokens result into only 1 response and all other responses are lost.

## Expected behavior

async calls should result into corresponding async responses, returning access token for each call.

## Minimal reproduction of the problem with instructions

Below code returns only 1 response instead of 2.

function getAccessToken(msalConfig){
   var userAgentApplication= new Msal.UserAgentApplication(msalConfig);
  userAgentApplication.acquireTokenSilent(tokenRequest).then(function (tokenResponse) {
        //control goes here only once.
        console.log("Token for " + tokenResponse.scopes[0] + " : " + tokenResponse.accessToken);
    }
}

getAccessToken(msalConfigWIthClientId1);
getAccessToken(msalConfigWIthClientId2);

**Comment by sameerag** [comment]
@zoperohan MSAL currently only supports one account => only one `clientId` is supported. `multiple accounts` supports is in the pipeline, unfortunately we do not have a specific date yet. 

Having said that, for the same `clientId` multiple tokens can be fetched concurrently. We have added more optimization on the same in our latest beta: `msal-1.2.0-beta.1` which is in npm now. We will be announcing the release notes today on the same. Please try the beta and let us know how it works for you.

**Comment by jmckennon** [comment]
Closing this issue, as it's been some time and I'm assuming the issue has been resolved. Feel free to open a new issue if you have further questions.

**Comment by sameerag** [comment]
Concurrent requests are now supported in the latest `msal js` `beta.4` in npm. Please test it and let us know if this is still an issue.
